app/controllers/general_reports_controller.rb

```diff
@@ -88,7 +88,7 @@ def init_report_params
   end
 
 
-  def init_report_headers(report_on_label)
+  def init_report_headers(report_on_label = nil)
```

Comment:
It doesn't get called without `report_on_label` yet.  But the `report_on_label` is optional.  I ran into this while experimenting with testing, before finding a better testing approach.